Tasks/RunLoadTest/CltTasksUtility.ps1

```diff
@@ -149,34 +149,74 @@ function isNumericValue ($str) {
 	return $isNum
 }
 
-function ValidateFiles($inputName, $fileName)
+function ValidateFiles($inputName, $loadtestDrop, $fileName, $testSettings)
 {
-	$file = Get-ChildItem -Path $TestDrop -recurse | where {$_.Name -eq $fileName} | Select -First 1
-	if ($file)
+	# Upgrade scenario start..
+	if (-Not([System.IO.Path]::IsPathRooted($loadtestDrop)))
 	{
-		# Check for fileName
-		$global:ScopedTestDrop = $file.Directory.FullName
-		Write-Host -NoNewline ("Selected {0} is '{1}' under '{2}'"  -f $inputName, $file.FullName, $global:ScopedTestDrop)
+		$loadtestDrop=[System.IO.Path]::Combine($env:SYSTEM_DEFAULTWORKINGDIRECTORY,$loadtestDrop);
+		Write-Host -NoNewline "Updated test drop location is $loadtestDrop";
+
+		if (-Not([string]::IsNullOrWhiteSpace($testSettings)) -and 
+		    -Not([System.IO.Path]::IsPathRooted($testSettings)))
+		{
+			$testSettings=[System.IO.Path]::Combine($env:SYSTEM_DEFAULTWORKINGDIRECTORY,$testSettings);
+			Write-Host -NoNewline "Updated test settings file is $testSettings";
+		}
```

Comment:
this doesnt seem to be related/dependent on the outer if block. it can come out and stay along with the test settings section.